setup.py

```diff
@@ -80,7 +80,7 @@
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
```

Comment:
As per the psycopg2 release notes (http://initd.org/psycopg/docs/news.html#what-s-new-in-psycopg-2-8), in 2.8 they stopped installing binary packages by default when you just ask for `psycopg2`. You now have to specify `psycopg2-binary` instead, otherwise the package is built from source. By `mro` depending on `psycopg2`, any consumers of the `mro` package then need to build it from source as well, which is unnecessary.